### DISCUSSION TITLE
Confine pointer to presentation display/window.

### DISCUSSION THREAD

**Discussion by SirLesP** [jahnf/Projecteur#218] [Ideas]
So far when using the Projecteur program I've found that the Projecteur pointer can be moved between the audience display (projector) and the presenter view, just as a normal mouse pointer can be.
However, in this case I'd like to 'lock' the pointer within the confines of the audience display. (I'd use another mouse or touchpad for interaction with the presenter view.)
I'm sure this can be done, but none of the methods I've tried so far actually work for me. (I've looked at xinput map-to-output, which appears to be non-functional, and transformation matrices with xinput set-prop and xdotools. I can make the mouse behave badly, but not do what I'm trying to do...)
It would be a lovely feature to have built in to Projecteur, though for now I'm looking for suggestions as to how to set this up directly in X or Wayland's settings.
Neil

**Comment by jahnf** [comment]
Interesting case, but at least right now I don't see this as part of _Projecteur_, as all of these devices merely act as a (generic) mouse and/or keyboard input. The overlay and the spotlight depend on the mouse position (current implementation)

If you would connect two different mice to your notebook - you would still only control the ONE mouse pointer with both devices. (Same with your touchpad + additional mouse)
